src/main/kotlin/no/nav/syfo/dialogmote/domain/MotedeltakerVarselType.kt

```diff
@@ -94,6 +94,15 @@ fun MotedeltakerVarselType.getDialogMeldingType(): DialogmeldingType {
     }
 }
 
+fun MotedeltakerVarselType.getDialogMeldingKodeverk(): DialogmeldingKodeverk {
+    return when (this) {
+        MotedeltakerVarselType.INNKALT -> DialogmeldingKodeverk.DIALOGMOTE
+        MotedeltakerVarselType.NYTT_TID_STED -> DialogmeldingKodeverk.DIALOGMOTE
+        MotedeltakerVarselType.AVLYST -> DialogmeldingKodeverk.HENVENDELSE
```

Comment:
Litt rart kanskje, men det er ikke dialogmøte-kodeverket som brukes for avlysning av og referat til dialogmøte.